draft-ietf-taps-transport-security.md

```diff
@@ -482,30 +485,33 @@ but this document considers them together, since that is the most common pattern
 
 #### IKEv2
 
-IKEv2 is a control protocol that runs on UDP port 500. Its primary goal is to generate keys for Security Associations (SAs).
-An SA contains shared (cryptographic) information used for establishing other SAs or keying ESP; See {{ESP}}.
-IKEv2 first uses a Diffie-Hellman key exchange to generate keys for the "IKE SA", which is a set of keys
-used to encrypt further IKEv2 messages. It then goes through a phase of authentication in which both peers
-present blobs signed by a shared secret or private key, after which another set of keys is derived, referred
-to as the "Child SA". These Child SA keys are used by ESP.
+IKEv2 is a control protocol that runs on UDP port 500 or TCP port 4500. (IKEv2 cannot run over
+UDP on any other port.) Its primary goal is to generate keys for Security Associations (SAs).
+An SA contains shared (cryptographic) information used for establishing other SAs or keying ESP;
+See {{ESP}}. IKEv2 first uses a Diffie-Hellman key exchange to generate keys for the "IKE SA",
+which is a set of keys used to encrypt further IKEv2 messages. IKE then performs a phase of
+authentication in which both peers present blobs signed by a shared secret or private key that
+authenticates the entire IKE exchange and the IKE identities. IKE then derives further sets of
+keys on demand, which together with traffic policies are referred to as the "Child SA". These
+Child SA keys are used by ESP.
 
 IKEv2 negotiates which protocols are acceptable to each peer for both the IKE and Child SAs using
-"Proposals". Each proposal may contain an encryption algorithm, an authentication algorithm, a
-Diffie-Hellman group, and (for IKE SAs only) a pseudorandom function algorithm. Each peer may
+"Proposals". Each proposal contains an encryption with authentication algorithm or an AEAD algorithm,
```

Comment:
Nit: This is a little confusing, maybe since I don't know IPsec all that well: is the "encryption with authentication algorithm" in lieu of AEAD something like AES with HMAC-SHA256, or is an AEAD algorithm required and "or an AEAD algorithm" is an apposition? Either way, I think something needs to be reworded.

Reply:
According to @tfpauly, it's one or the other! Thanks for pointing this out. I'll rephrase it to:

“Each proposal specifies an encryption and authentication algorithm, or an AEAD algorithm, ...”